src/public/js/app.js

```diff
@@ -209058,7 +209058,7 @@ function (_ModelPipe) {
       }).map(function (attribute) {
         return _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].singleQuotePad(attribute.name) + " => $this->" + attribute.name;
       }).concat(model.relationships.hasMany.concat(model.relationships.belongsToMany).map(function (target) {
-        return _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].singleQuotePad(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].snakeCase(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pluralize(target.name))) + " => " + _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pascalCase(target.name) + "Resource::collection($this->whenLoaded(" + _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].singleQuotePad(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].snakeCase(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pluralize(target.name))) + "))";
+        return _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].singleQuotePad(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].snakeCase(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pluralize(target.name))) + " => new " + _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pascalCase(target.name) + "Collection($this->whenLoaded(" + _utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].singleQuotePad(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].snakeCase(_utilities_Formatter__WEBPACK_IMPORTED_MODULE_2__["default"].pluralize(target.name))) + "))";
```

Comment:
Would be nice if we exclude all build files to reduce diff noises when reviewing PR.

I am not too aware how to tell composer to include this on package release. 
A reference on how this would be done in npm package: https://stackoverflow.com/questions/31642477/how-to-publish-a-npm-package-with-distribution-files

Reply:
Yes, this would be really nice!!